test/parallel/test-timers-setunreftimeout.js

```diff
@@ -0,0 +1,71 @@
+// Flags: --expose-internals
+
+'use strict';
+
+const common = require('../common');
+
+const { strictEqual } = require('assert');
+const { setUnrefTimeout } = require('internal/timers');
+
+// ERR_INVALID_CALLBACK
+{
+  // It throws ERR_INVALID_CALLBACK when first argument is not a function.
+  common.expectsError(
+    () => setUnrefTimeout(),
+    {
+      code: 'ERR_INVALID_CALLBACK',
+      type: TypeError,
+      message: 'Callback must be a function'
+    }
+  );
+}
+
+{
+  let called = false;
+  const timer = setUnrefTimeout(common.mustCall(() => {
+    called = true;
+  }), 1);
+
+  const testTimer = setUnrefTimeout(common.mustCall(() => {
+    strictEqual(called, false, 'unref pooled timer returned before check');
```

Comment:
Instead of an extra variable and checking `called`, you can just use `common.mustNotCall` here and then `clearTimeout(testTimer);` inside the first `setUnrefTimeout` (the one above). That will tell you that the first timer either ran and worked just fine or it didn't run and then the `testTimer` will also execute and fail.

Reply:
Thanks! I understood. I'll fix it.